# Issue 33406: Automatic image build is broken on GitLab

archive/issues_033169.json:
```json
{
    "body": "CC:  @videlec\n\nCurrently, images for https://hub.docker.com/repository/docker/sagemath/sagemath are not built automatically anymore. The main issue seems to be that there are no dedicated runners registered there anymore.\n\nThere are likely other problems since the [GitLab](GitLab) Pipeline has not been running for ages now.\n\nIssue created by migration from https://trac.sagemath.org/ticket/33406\n\n",
    "created_at": "2022-02-23T18:49:08Z",
    "labels": [
        "component: docker",
        "bug"
    ],
    "milestone": "https://github.com/sagemath/sagetest/milestones/sage-duplicate/invalid/wontfix",
    "title": "Automatic image build is broken on GitLab",
    "type": "issue",
    "url": "https://github.com/sagemath/sagetest/issues/33406",
    "user": "https://github.com/saraedum"
}
```
CC:  @videlec

Currently, images for https://hub.docker.com/repository/docker/sagemath/sagemath are not built automatically anymore. The main issue seems to be that there are no dedicated runners registered there anymore.

There are likely other problems since the [GitLab](GitLab) Pipeline has not been running for ages now.

Issue created by migration from https://trac.sagemath.org/ticket/33406





---

archive/issue_comments_471920.json:
```json
{
    "body": "(I am not planning to do anything about this at the moment. There does not seem to be a real interest in using [GitLab](GitLab) for [SageMath](SageMath) development. I just needed an issue to reference in the README on Docker Hub.)",
    "created_at": "2022-02-23T18:49:54Z",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "type": "issue_comment",
    "url": "https://github.com/sagemath/sagetest/issues/33406#issuecomment-471920",
    "user": "https://github.com/saraedum"
}
```

(I am not planning to do anything about this at the moment. There does not seem to be a real interest in using [GitLab](GitLab) for [SageMath](SageMath) development. I just needed an issue to reference in the README on Docker Hub.)



---

archive/issue_events_088096.json:
```json
{
    "actor": "https://github.com/mkoeppe",
    "created_at": "2022-05-03T15:17:06Z",
    "event": "milestoned",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "milestone": "sage-9.7",
    "type": "issue_event",
    "url": "https://github.com/sagemath/sagetest/issues/33406#event-88096"
}
```



---

archive/issue_comments_471921.json:
```json
{
    "body": "Changing status from new to needs_review.",
    "created_at": "2022-09-10T16:46:40Z",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "type": "issue_comment",
    "url": "https://github.com/sagemath/sagetest/issues/33406#issuecomment-471921",
    "user": "https://github.com/mkoeppe"
}
```

Changing status from new to needs_review.



---

archive/issue_comments_471922.json:
```json
{
    "body": "Sounds like we can close it as wontfix",
    "created_at": "2022-09-10T16:46:40Z",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "type": "issue_comment",
    "url": "https://github.com/sagemath/sagetest/issues/33406#issuecomment-471922",
    "user": "https://github.com/mkoeppe"
}
```

Sounds like we can close it as wontfix



---

archive/issue_events_088097.json:
```json
{
    "actor": "https://github.com/mkoeppe",
    "created_at": "2022-09-10T16:46:40Z",
    "event": "demilestoned",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "milestone": "sage-9.7",
    "type": "issue_event",
    "url": "https://github.com/sagemath/sagetest/issues/33406#event-88097"
}
```



---

archive/issue_events_088098.json:
```json
{
    "actor": "https://github.com/mkoeppe",
    "created_at": "2022-09-10T16:46:40Z",
    "event": "milestoned",
    "issue": "https://github.com/sagemath/sagetest/issues/33406",
    "milestone": "sage-duplicate/invalid/wontfix",
    "type": "issue_event",
    "url": "https://github.com/sagemath/sagetest/issues/33406#event-88098"
}
```
